internal/backend/db/dbgorm/orgs.go

```diff
@@ -14,21 +14,25 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-func (gdb *gormdb) GetOrg(ctx context.Context, oid sdktypes.OrgID) (sdktypes.Org, error) {
+func (gdb *gormdb) GetOrg(ctx context.Context, oid sdktypes.OrgID, n sdktypes.Symbol) (sdktypes.Org, error) {
 	if oid == authusers.DefaultOrg.ID() {
 		return authusers.DefaultOrg, nil
 	}
 
 	q := gdb.db.WithContext(ctx)
 
-	if !oid.IsValid() {
-		return sdktypes.InvalidOrg, sdkerrors.NewInvalidArgumentError("missing id")
+	if !oid.IsValid() && !n.IsValid() {
+		return sdktypes.InvalidOrg, sdkerrors.NewInvalidArgumentError("missing id or name")
 	}
 
 	if oid.IsValid() {
 		q = q.Where("org_id = ?", oid.UUIDValue())
 	}
 
+	if n.IsValid() {
```

Comment:
Shouldn't this be "else"? What happens if the user specifies non-matching org ID and name?

Reply:
fixed

Reply:
actually my mistake
if a user specifies non matching, it'll just be not found.